src/.env/dev/terraform.tfvars

```diff
@@ -0,0 +1,80 @@
+# general
+env_short      = "d"
+env            = "dev"
+prefix         = "dvopla"
```

Comment:
why not devopslab?

Reply:
because we need to use prefix that use only 6 charters. Some resources have a limits in the name